rosidl_generator_py/cmake/rosidl_generator_py_generate_interfaces.cmake

```diff
@@ -76,17 +79,17 @@ endforeach()
 
 if(NOT "${_generated_msg_py_files} " STREQUAL " ")
   list(GET _generated_msg_py_files 0 _msg_file)
-  get_filename_component(_parent_folder "${_msg_file}" DIRECTORY)
+  get_filename_component(_parent_folder2 "${_msg_file}" DIRECTORY)
   list(APPEND _generated_msg_py_files
-    "${_parent_folder}/__init__.py"
+    "${_parent_folder2}/__init__.py"
```

Comment:
I added them because in one case parent_folder is a full path and in the other case it's just the name and in the other case it was the path of the parent folder. Before esteve was not looping over the type support so he could afford to rewrite the variable. But you're right the variable name is confusing and I will rename them to something more explicit